src/PhpSpreadsheet/Shared/StringHelper.php

```diff
@@ -688,4 +693,17 @@ public static function testStringAsNumeric($textValue)
 
         return (is_numeric(substr($textValue, 0, strlen($v)))) ? $v : $textValue;
     }
+
+    /**
+     * If text exceed chars limit in cell, it is truncated.
+     */
+    public static function applyCharsLimit(string $text): string
+    {
+        if (strlen($text) > self::CELL_CHARS_LIMIT) {
```

Comment:
Is the limit characters, or is it bytes? `strlen()` measures the string in bytes; the PhpSpreadsheet `StringHelper::countCharacters()` method returns the character length